openvpn.opam

```diff
@@ -16,8 +16,8 @@ build: [
 ]
 
 depends: [
-  "dune" {build}
-  "ocaml" {>= "4.07.0"}
+  "dune"
+  "ocaml" {>= "4.08.0"}
```

Comment:
Why do we need to leave 4.07 behind for this changeset?

Reply:
`Fun.protect`.